src/test/java/homework_2/random_chars_table/RandomCharsTableTest.java

```diff
@@ -0,0 +1,68 @@
+package homework_2.random_chars_table;
+
+import base.UnitBase;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RandomCharsTableTest extends UnitBase {
+
+    @Test
+    void given_first_bad() {
```

Comment:
Opt: better if your test cases names follow given_when_then pattern, when can be omitted. Something like givenInvalid_thenRuntimeException